scripts/automation/Radius/Functions/Public/Get-JCRCertReport.ps1

```diff
@@ -0,0 +1,76 @@
+function Get-JCRCertReport {
+    param(
+        [Parameter(Mandatory)]
+        [ValidateScript({
+                $directory = Split-Path -Path $_ -Parent
+                if (-not (Test-Path -Path $directory -PathType Container)) {
+                    throw "The directory '$directory' does not exist."
+                }
+                if (-not ($_ -like '*.csv')) {
+                    throw "The specified path '$_' does not end with '.csv'."
+                }
+                return $true
+            })]
+        [System.IO.FileInfo]$ExportFilePath
+    )
+
```

Comment:
If I'm going to be nitpicky, I'd say there really should be begin/ process/ end blocks here. I don't have anything else to add to the function